near-sdk/src/json_types/integers.rs

```diff
@@ -80,6 +82,7 @@ mod tests {
         test_serde!(U128, u128, 10u128.pow(18));
         test_serde!(U128, u128, 2u128.pow(100));
         test_serde!(U128, u128, u128::max_value());
+        assert!(U128::from(u128::min_value()) < U128::from(u128::max_value()));
```

Comment:
I don't think these tests are really necessary for things like this, and only test `PartialOrd`, but I think it's fine to keep!